src/core/services/gesture/gesture.js

```diff
@@ -318,6 +331,19 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
       }
     });
 
+  function getTouchAction() {
+    var testEl = angular.element('<div>')[0];
+    var vendorPrefixes = ['', 'webkit', 'Moz', 'MS', 'ms', 'o'];
```

Comment:
[According to Caniuse](http://caniuse.com/#search=touch-action), there's only the `ms` vendor prefix.